### PR TITLE
BaSP-MR-69: List a table of super admins and delete petition functionality

### PR DESCRIPTION
**Pull request by IvanJukonis**
Hello, I have submitted the PR for ticket number 69. This ticket required the development of a table listing the Super Admin component for its resolution. Each row corresponding to a superadmin should contain a delete button (with its corresponding functionality) and an edit button (without functionality). Finally, the creation of a button to add a super admin is required, as well as the creation of confirmation modals for the delete function.

**All the requirements of the ticket have been resolved.**

### CODE REVIEW

**Comment by IvanJukonis** [comment]
[SUPER ADMIN - List/delete table](https://trello.com/c/C8r6fRY6/69-super-admin-list-delete-table)

**Review by CristianLotorto — CHANGES_REQUESTED** [review]
there's an issue in delete functionality

**Comment by IvanJukonis** [comment]
> there's an issue in delete functionality

The issue is not found in the delete but in the add functionality. This has already been discussed with Cristian and we reached the agreement that the error will be mentioned in my next PR, where a solution will be sought. Although the delete functionality may be affected by this error, there is nothing to correct.

**Review by CristianLotorto — APPROVED** [review]

**Review by maurojjzz — APPROVED** [review]
everything looks great

**Review by NicoCagnina — CHANGES_REQUESTED** [review]

File: src/Components/SuperAdmins/index.js
[the remark this answers is not in the document]
Comment (IvanJukonis) [reply src/Components/SuperAdmins/index.js:1]: Oh i did the same on my other PR, I wanted to change the JS termination of the file into JSX.

**Review by cbaleani — APPROVED** [review]
Looks great!

**Pull request merged**